src/test/java/core/basesyntax/dao/FruitDaoImplTest.java

```diff
@@ -0,0 +1,39 @@
+package core.basesyntax.dao;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.Fruit;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+public class FruitDaoImplTest {
+    private static final Fruit TEST_FRUIT = new Fruit("carrot", 25);
+    private static FruitDao fruitDao;
+
+    @Before
+    public void setUp() throws Exception {
+        fruitDao = new FruitDaoImpl();
```

Comment:
better to do it in BeforeClass
same everywhere

Reply:
I intended to do it in BeforeClass everywhere, I don't understand how it happened that in some classes it turned into Before.
Of course I'm fixing this issue